### PR TITLE
Don't translate pane and dashlet names in configs

### LINKED ISSUE #3542: Changes in dashboard.ini needed due to updated german localization
As there is a updated german localization fpr IcingaWeb2 2.6.0 existing dashboard.ini files for users need to be updated to use the new words.
The default dashboard "Aktuelle Incidents" was renamed to "Aktuelle Vorfälle".

I have fixed this for my using "vi" and ```:%s/Incidents/Vorfälle/g``` 

Maybe add a shot mention to the update docs?

### PR DESCRIPTION
**Pull request by nilmerg**
Introduces a major breaking change. All non-english users need to adjust their dashboard.ini after this change. (In case they've adjusted dashlets provided by modules)

Missing:
- [x] Upgrade documentation

fixes #3542

### CODE REVIEW

File: library/Icinga/Web/Widget/Dashboard.php [hunk 1]
```diff
@@ -77,15 +77,15 @@ public function load()
         foreach ($navigation as $dashboardPane) {
             /** @var DashboardPane $dashboardPane */
             $pane = new Pane($dashboardPane->getLabel());
```
Comment (lippserd) [inline library/Icinga/Web/Widget/Dashboard.php]: My quick fix for this also included the following here:
```
$pane = (new Pane($dashboardPane->getName()))->setTitle($dashboardPane->getLabel());
```
Maybe this helps? With this change only, panes won't get translated upon saving.

Comment (nilmerg) [reply library/Icinga/Web/Widget/Dashboard.php]: Tried it, doesn't affect dashlets though. And honestly, my solution works. Maybe there's a simpler one, but any additional second used to look at this is a wasted one.

File: doc/80-Upgrading.md [hunk 2]
```diff
@@ -6,6 +6,14 @@ Icinga Web 2 database.
 Specific version upgrades are described below. Please note that version updates are incremental. An upgrade from
 v2.3 to v2.5 requires to follow the instructions for v2.4 too.
 
+## Upgrading to Icinga Web 2 2.7.x <a id="upgrading-to-2.7.x"></a>
+
+**Breaking changes**
+
+* If you're using a language other than english and you've adjusted or disabled module dashboards, you'll need to
+  update all of your `dashboard.ini` files. Either replace all section names with their english equivalent or just
+  remove the sections (Though, only those for module dashboards!) and apply all changes again.
```
Comment (dnsmichi) [inline doc/80-Upgrading.md]: Maybe we can add an example here - old vs new :)

**Comment by lippserd** [comment]
We should provide a migrate command for this instead of having users to do this on their own.

**Pull request merged**